src/routes/auth/auth.controller.spec.ts

```diff
@@ -176,37 +185,34 @@ describe('AuthController', () => {
           message,
           signature,
         })
-        .expect(422)
+        .expect(401)
         .expect(({ headers, body }) => {
           expect(headers['set-cookie']).toBe(undefined);
 
           expect(body).toStrictEqual({
-            code: 'custom',
-            message: 'Must be within 900 seconds',
-            path: ['message', 'expirationTime'],
-            statusCode: 422,
+            message: 'Unauthorized',
+            statusCode: 401,
           });
         });
-      // Nonce not deleted
-      await expect(cacheService.get(cacheDir)).resolves.toBe(
-        nonceResponse.body.nonce,
-      );
+      // Nonce deleted
```

Comment:
The nonce wasn't deleted before this change because this validation failed when the `ValidationPipe(VerifyAuthMessageDtoSchema)` was invoked at the route level, without cache access, right? (I think this _side-effect_ of the change makes total sense, I'm just trying to confirm my reasoning 🙂)

Reply:
Correct. I think it makes more sense having moved it.

Reply:
Absolutely! Thanks!